gordo_components/builder/mlflow_utils.py

```diff
@@ -398,48 +401,36 @@ def get_spauth_kwargs() -> dict:
     )
 
 
-class DatetimeEncoder(json.JSONEncoder):
+class MetadataEncoder(json.JSONEncoder):
     """
     Encode datetime.datetime objects as strings
 
     Example
     -------
-    >>> s = json.dumps({"now":datetime.now(tz=UTC)}, cls=DatetimeEncoder, indent=4)
+    >>> s = json.dumps({"now":datetime.now(tz=UTC)}, cls=MetadataEncoder, indent=4)
     >>> s = '{"now": "2019-11-22 08:34:41.636356+"}'
     """
 
+    @staticmethod
+    def is_int_or_float(obj):
+        return any([type_str in str(type(obj)) for type_str in ("float", "int")])
+
+    @staticmethod
+    def reduce_bit_type(obj):
+        for type_cls, type_str in [(float, "float"), (int, "int")]:
+            if type_str in str(type(obj)):
+                return type_cls(obj)
+
     def default(self, obj):
         if isinstance(obj, datetime):
             return obj.strftime("%Y-%m-%d %H:%M:%S.%f+%z")
+        # Typecast floats and ints to 8bit python number objects
```

Comment:
This is just wrong. You're saying you want to cast the number to 8bits, using python's `int` or `float`? 8 bits is 0-255 for unsigned and -128-127 for signed; definitely not what `int` or `float` is.

Reply:
Ok... ok.